tests/test_response.py

```diff
@@ -325,6 +342,66 @@ def test_repr(json_data, expected_repr):
     assert repr(response) == expected_repr
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 10),
+    reason=(
+        "pprint for dataclasses is only available since 3.10+, "
+        "https://bugs.python.org/issue43080"
+    ),
+)
+@pytest.mark.parametrize(
+    "json_data, expected",
+    [
+        # A "short" response's pprint looks the same as its repr.
+        ({"foo": {"barBaz": 456}}, "Response(foo=(bar_baz=456))"),
+        # A "long" response's pprint triggers line wrapping in pretty-printing.
+        (
+            {ascii_lowercase[i]: i for i in range(15)},
+            "Response(a=0,\n"
+            "         b=1,\n"
+            "         c=2,\n"
+            "         d=3,\n"
+            "         e=4,\n"
+            "         f=5,\n"
+            "         g=6,\n"
+            "         h=7,\n"
+            "         i=8,\n"
+            "         j=9,\n"
+            "         k=10,\n"
+            "         l=11,\n"
+            "         m=12,\n"
+            "         n=13,\n"
+            "         o=14)",
+        ),
+        (
+            {
+                "foo": {ascii_lowercase[i]: i for i in range(3)},
+                "fooBar": {ascii_lowercase[i]: i for i in range(15)},
+            },
+            "Response(foo=(a=0, b=1, c=2),\n"
+            "         foo_bar=(a=0,\n"
+            "                  b=1,\n"
+            "                  c=2,\n"
```

Comment:
Another doc note: In a future PR, maybe somewhere in the Sphinx docs we could encourage people to use either IPython or Jupyter notebooks for syntax highlighting and nicer pretty prints like this (the regular Python interactive interpreter or directly calling `print(response)` would have this rendered in one single unwieldy long line).